src/toil_lib/tools/variant_annotation.py

```diff
@@ -0,0 +1,106 @@
+import os
+import tarfile
+
+from toil_lib.programs import docker_call
+
+
+def gatk_genotype_gvcfs(job, gvcf_ids, ref, fai, ref_dict, annotations=None, emit_threshold=10.0,
```

Comment:
Nit: I would split out these arguments onto newlines that are logically grouped. E.g., I would have the job on one line, filestore IDs on one line, thresholds on another, etc.